app/controllers/chargeback_assignment_controller.rb

```diff
@@ -154,21 +122,22 @@ def cb_assign_set_record_vars
     end
   end
 
-  # Set form variables for edit
-  def cb_assign_set_form_vars
+  # Set form variables for edit, once user has made type selection
+  def set_form_vars
     @edit = {
       :cb_rates  => {},
       :cb_assign => {},
     }
+    @edit[:new]     = HashWithIndifferentAccess.new
+    @edit[:current] = HashWithIndifferentAccess.new
+    @edit[:new][:type] = params[:id] || @tabform
```

Comment:
Maybe we want to sanitize this, by matching if the value is among `ChargebackRate::VALID_CB_RATE_TYPES`